MediaBrowser.Providers/Music/MusicBrainzAlbumProvider.cs

```diff
@@ -714,34 +714,32 @@ private string GetFirstReleaseGroupId(XmlReader reader)
             return null;
         }
 
-        internal Task<HttpResponseInfo> GetMusicBrainzResponse(string url, bool isSearch, CancellationToken cancellationToken)
-        {
-            return GetMusicBrainzResponse(url, isSearch, false, cancellationToken);
-        }
-
         /// <summary>
-        /// Gets the music brainz response.
+        /// Makes request to MusicBrainz server and awaits a response.
         /// </summary>
-        internal async Task<HttpResponseInfo> GetMusicBrainzResponse(string url, bool isSearch, bool forceMusicBrainzProper, CancellationToken cancellationToken)
+        internal async Task<HttpResponseInfo> GetMusicBrainzResponse(string url, CancellationToken cancellationToken)
         {
-            var urlInfo = new MbzUrl(MusicBrainzBaseUrl, 1000);
-            var throttleMs = urlInfo.throttleMs;
+            // The Jellyfin user-agent is unrestricted but source IP must not exceed
+            // one request per second, therefore we rate limit to avoid throttling
+            // https://musicbrainz.org/doc/XML_Web_Service/Rate_Limiting
+            const long QueryIntervalMs = 1000u;
```

Comment:
I guess this should be class-level const, not stuffed into some internal method.
Also it might be prudent to set the interval to, say, `1050 ms` just to be on the safe side.

Reply:
Moved to class scope and increased to 1050.